journeys/journeys-service.yaml

```diff
@@ -0,0 +1,197 @@
+openapi: "3.0.1"
+info:
+  title: "SCALE Journeys Service"
+  version: v0.0.1
+  description: This API allows UI's to define journeys associated to a unique identifier supplied. This is used to store the state of progress within the journey. The UI is responsible for supplying a unique identifier within the bounds of itself and the user.
+#  PLACEHOLDERs - to be uncommented when implemented
+  termsOfService: "http://api.crowncommercial.gov.uk/terms/"
+  contact:
+    email: "apiteam@crowncommercial.gov.uk"
+  license:
+    name: "Open Government Licence 3.0"
+    url: "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+#
+#
+#security:
+#  - OAuth2:
+#    - read
+paths:
+  /journeys:
+    post:
+      description: Creates a new journey defining the number of states and their initial state.
+      tags:
+        - Journeys
+      operationId: create-journey
+      requestBody:
+        content:
+          application/json::
+            schema:
+              $ref: '#/components/schemas/Journey'
+
+      responses:
+        '201':
+            description: TeamID for the newly created Team
+            content:
+                application/json:
+                    schema:
+                        $ref: '#/components/schemas/JourneyId'
+        '401':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/401Unauthorized'
+        '403':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/403Forbidden'
+        '429':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/429TooManyRequests'
+        '500':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/500InternalServerError'
+        '502':
+          $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OpenAPI_Standards/CCS_Standard_API_Responses.yaml#/components/responses/502BadGateway'
+      summary: Create a new Journey
+      
+      
+  /journeys/{journey-id}/steps:
+    get:
+      summary: Return current Journey state.
+      description: Retrieves a list of states for each step within the process. 
+      tags:
+        - Journeys
+      operationId: get-journey-state
+      parameters:
+        - $ref: '#/components/parameters/JourneyParam'
+      responses:
+        200:
+          description: Journey State
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JourneyState'
+        '400':
```

Comment:
Can we limit the error responses to only those that can / will be returned by each endpoint?  There are a number here that will never be returned, could confuse the test team.

Reply:
Removed where I think, hope its right